coq-core.opam

```diff
@@ -27,7 +27,7 @@ homepage: "https://coq.inria.fr/"
 doc: "https://coq.github.io/doc/"
 bug-reports: "https://github.com/coq/coq/issues"
 depends: [
-  "dune" {>= "2.9"}
+  "dune" {>= "3.6" & >= "3.6.1"}
```

Comment:
Why is it written like this instead of just `>= "3.6"`?

Reply:
As we declare `(lang dune 3.6)` dune adds automatically the `dune >= 3.6` lower bound when auto-generating the opam file, then our bound is added, as 3.6.0 contained a bug fatal to Coq.